### ISSUE TITLE
Install error: ImportError: No module named ldtp

### ISSUE THREAD

**Issue by richelbilderbeek** [ldtp/ldtp2#37]
As can be seen from [a Travis CI build log](https://travis-ci.org/richelbilderbeek/ldtp2/builds/226728780#L804), importing `ldtp` fails:

```
python minimal.py
```

gives:

```
Traceback (most recent call last):
  File "minimal.py", line 3, in <module>
    from ldtp import *
ImportError: No module named ldtp
```

First line in `minimal.py` is `from ldtp import *` (it is a script suggested by @nagappan at #35).

This is the compete reproduction of it ([from the .travis.yml script](https://github.com/richelbilderbeek/ldtp2/blob/master/.travis.yml)):

```
# Install prerequisites
sudo apt-get install python-twisted-web
sudo apt-get install python-wnck
# Build LDTP
python setup.py build
sudo python setup.py install
# Build and run examples
cd examples
python minimal.py
```

It guess someone with some Python knowledge can easily see what went wrong.

**Comment by nagappan** [comment]
This means ldtp is not installed in path, check /usr/local/lib/python folder (I think)

**Comment by richelbilderbeek** [comment]
Does this mean that [setup.py](https://github.com/ldtp/ldtp2/blob/master/setup.py) does not install LDTP correctly?

**Comment by stuaxo** [comment]
Are you sure that when it runs "python", are you sure it is the same instance of python that ldtp is installed to ?

Try using the full path to the python interpreter, since the first one in the path may not be the same one.

**Comment by stuaxo** [comment]
You can output see python is being called by adding this somewhere in your travis script

```shell
echo `which python`
```

**Comment by stuaxo** [comment]
It looks like you may have yet another python installed in /usr/local.

I would specify the full path to python and pip when you use them to avoid ambiguity.

Without doing this you might get different copies especially when using sudo.

**Comment by richelbilderbeek** [comment]
I've used `usr/local/python` instead of just `python`. From [the build log](https://travis-ci.org/richelbilderbeek/ldtp2/builds/487444481#L1053), I now get:

```
/usr/bin/python
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/ldtpd/__init__.py", line 38, in <module>
    from xmlrpc_daemon import XMLRPCLdtpd
  File "/usr/local/lib/python2.7/dist-packages/ldtpd/xmlrpc_daemon.py", line 25, in <module>
    import core
  File "/usr/local/lib/python2.7/dist-packages/ldtpd/core.py", line 24, in <module>
    from pyatspi import findDescendant, Registry
ImportError: No module named pyatspi
```

I assume that stops even getting to importing ldtp...

**Comment by richelbilderbeek** [comment]
Yes, I understand.

I give up. If I cannot get such a simple example running working on Travis, I will abandon ldtp until someone does succeed. I hope this Issue log helps him/her to do so.

Closing the Issue...

**Comment by stuaxo** [comment]
Yeah - I found it very hard to get it working, it seems like the it's a very old library and built as such.
I also gave up as I was only playing with it on a whim.


Like other libraries that came out before virtualenv existed it doesn't work well with it (or at all without a lot of effort), the code style looks very like C translated into python.


The best thing that could happen is to make a new library with the same license, and pythonic code style, using ldtp for reference to build it.
